### DISCUSSION TITLE
Equation with cases

### DISCUSSION THREAD

**Discussion by seghier** [ForNeVeR/wpf-math#410] [Q&A]
Hello
How to create this equation? 

```
\documentclass{article}
\usepackage{amssymb}
\usepackage[overload]{empheq}
\newcommand{\for}{\text{for }}

\begin{document}

\begin{equation*}[right=\empheqrbrace{=xy.}]
    X(m,n)=
    \begin{cases}
      x(n),& \for 0\leqslant n \leqslant 1 \\
      x(n-1),& \for 0\leqslant n \leqslant 1 \\
      x(n-1),& \for 0\leqslant n \leqslant 1
    \end{cases}
\end{equation*}

\end{document} 
```

![0BzZH](https://github.com/ForNeVeR/xaml-math/assets/6026588/887c3e90-032a-4b05-bf5c-63bbea9f1d6a)

**Comment by ForNeVeR** [comment]
Try this:
```latex
X(m,n) =
\left.
    \cases{
        x(n),     & for\thickspace0 \leqslant n \leqslant 1 \\
        x(n - 1), & for\thickspace0 \leqslant n \leqslant 1 \\
        x(n - 1), & for\thickspace0 \leqslant n \leqslant 1
    }
\right\}

= xy.
```

<img width="323" alt="image" src="https://github.com/ForNeVeR/xaml-math/assets/92793/7efa0b15-8480-4707-adc3-7185b5ebcf65">

**Reply by seghier** [reply]
I got this error message:

`Error when parsing formula: Unknown symbol or command or predefined TeXFormula: 'leqslant'`

**Reply by ForNeVeR** [reply]
Ah, perhaps that's because `\leqslant` was only added recently. I'll publish a new version with its support today.

**Reply by seghier** [reply]
Thank you , waiting for it.
I notice there is difference between Latex formula and yours, is there a documentation showing all differences?
